### PR TITLE
Sensor Documentation Update

### PR DESCRIPTION
**Pull request by luis-camero**
1. Split up the `sensors.mdx` into multiple files 
2. Added references to driver source for each sensor.
3. Added instructions/requirements for each sensor.
4. Added documentation to create and build workspace.
5. Added documentation to create and setup customization package manually.

### CODE REVIEW

File: docs_versioned_docs/version-ros2humble/components/yaml/sensors/flir_blackfly.mdx [hunk 1]
```diff
@@ -20,38 +20,57 @@ camera:
...
-        chunk_enable_gain: true
-        chunk_selector_timestamp: Timestamp
-        chunk_enable_timestamp: true
 ```
 </td>
 </tr>
 </table>
+
+#### Package and Setup
+The Flir Blackfly cameras use the `spinnaker_camera_driver` ROS 2 package. The driver is open source, maintained by the ROS community, and hosted on [GitHub](https://github.com/ros-drivers/flir_camera_driver). The `spinnaker_camera_driver` depends on the [Spinnaker SDK](https://www.flir.ca/products/spinnaker-sdk/), which is installed alongside the package.
```
Comment (tonybaltovski) [inline docs_versioned_docs/version-ros2humble/components/yaml/sensors/flir_blackfly.mdx]: Consider saying `which is installed automatically alongside the package.`

File: docs_versioned_docs/version-ros2humble/ros/config/yaml/sensors/lidar3d.mdx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+title: 3D Lidar
+sidebar_label: 3D Lidar
+sidebar_position: 5
+toc_min_heading_level: 2
+toc_max_heading_level: 4
+---
+import VelodyneLidar from "/docs_versioned_docs/version-ros2humble/components/yaml/sensors/velodyne_lidar.mdx";
+
```
Comment (tonybaltovski) [inline docs_versioned_docs/version-ros2humble/ros/config/yaml/sensors/lidar3d.mdx:9]: Do we want some details what a 3D Lidar is like the rest?

File: docs_versioned_docs/version-ros2humble/ros/config/yaml/sensors/cameras.mdx [hunk 3]
```diff
@@ -46,22 +28,22 @@ For example, the color image topic will have multiple versions:
...
 sudo apt install ros-humble-clearpath-sensors
 ```
 
-#### Compressed
+### Compressed
 Compressed image types are not serialized. These are images compressed using the JPEG format at 80% quality.
 
 Use the launch file `image_raw_to_compressed.launch.py` to compress `sensor_msgs/Image` messages into compressed messages.
-
+therefore,
```
Comment (tonybaltovski) [inline docs_versioned_docs/version-ros2humble/ros/config/yaml/sensors/cameras.mdx]: Typo?

File: docs_versioned_docs/version-ros2humble/components/yaml/sensors/flir_blackfly.mdx (hunk 1 above)
Comment (luis-camero) [reply docs_versioned_docs/version-ros2humble/components/yaml/sensors/flir_blackfly.mdx]: Done

**Review by tonybaltovski — APPROVED** [review]

**Pull request merged**